src/taipy/core/scenario/scenario.py

```diff
@@ -300,6 +300,9 @@ def submit(
                 asynchronous mode.
             timeout (Union[float, int]): The optional maximum number of seconds to wait for the jobs to be finished
                 before returning.
+        Returns:
+            A dictionary, with `PipelineId^` as the key and a list of created `Job^` corresponding
```

Comment:
Well no not exactly, since when we submit a scenario, it also means we're submitting multiple pipelines, and as the current implementation puts it, we need a way to separate the jobs based on the pipeline/pipelineid. Hence, we are return a dictionary with pipelineId as key and list of jobs as values 😃 ofcource we can put all jobs into a single list, but it'll be a hassle trying to differentiate which pipeline it belongs to (going through 2 layers of parents: job -> task -> pipeline isn't particularly fun).

Reply:
Despite that, I agree that having some consistency would be so much better, just not sure how we can keep the consistency with pipeline submission in this case

Reply:
This has been changed to `A list of created `Job^`s.`